tests/telemetry/conftest.py

```diff
@@ -171,6 +171,10 @@ def test_eventd_healthy(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost, set
 
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
 
+    status = duthost.get_feature_status()
+    if 'eventd' not in status or status['eventd'] == 'disabled':
```

Comment:
And one more thing, if you want to use the return value as a tuple for get_feature_status, it's better to modify the name, not use 'status' as a return name. 
status['eventd'] should not be the correct value that you want to get. and would raise an exception.